cluster/manifests/polarsignals/02-rbac.yaml

```diff
@@ -0,0 +1,55 @@
+{{ if eq .Cluster.ConfigItems.polarsignals_enabled "true" }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/component: continuous-profiler
+    app.kubernetes.io/instance: polarsignals-agent
+    app.kubernetes.io/name: polarsignals-agent
+  name: polarsignals-agent
+  namespace: polarsignals
```

Comment:
This service account should be defined in admission-controller config, to allow it to run privileged: Similar like we do here: https://github.com/zalando-incubator/kubernetes-on-aws/blob/a70f7d7e2298ea32288ff4c28605798b70aa8272/cluster/manifests/02-admission-control/config.yaml#L115-L117

Reply:
thanks, done